Server/service/pingService.js

```diff
@@ -0,0 +1,74 @@
+const { Queue, Worker } = require("bullmq");
+const logger = require("../utils/logger");
+require("dotenv").config();
+
+const connection = {
+  host: process.env.BULL_MQ_HOST,
+  port: process.env.BULL_MQ_PORT,
+};
+
+const JOBS_PER_WORKER = 5;
+
+let queue;
+let workers = [];
+
+const enqueueJob = async (queue, monitor) => {
+  await queue.add("monitors", { monitor }, { delay: monitor.interval * 1000 });
+};
+
+const pingMonitor = (monitor) => {
+  console.log(`{url: ${monitor.url},`, `interval: ${monitor.interval}}`);
+};
+
+const createWorker = (queue) => {
+  return new Worker(
+    "monitors",
+    async (job) => {
+      //Ping the monitor and enqueue the job again to constantly monitor
+      pingMonitor(job.data.monitor);
+      enqueueJob(queue, job.data.monitor);
+    },
+    {
+      connection,
+    }
+  );
+};
+
+const startPingService = async (db) => {
+  queue = new Queue("monitors", {
+    connection,
+  });
+
+  const monitors = await db.getAllMonitors();
```

Comment:
@popnfresh234 Are we returning only `active` monitors here?

Reply:
Good point, I'm not sure if there's going to be a case where we need to return *all* montiors, maybe we only ever need to return only active monitors?

Reply:
I think in this case, we will only need active ones.